src/tools/tests/plan-generator-tests.ts

```diff
@@ -27,5 +32,56 @@ describe('recipe2plan', () => {
 
       assert.notInclude(actual, 'import arcs.core.data.*');
     });
+    it('creates valid types that refer to registered Schemas', async () => {
+      const manifest = await Manifest.parse(`\
+     particle A
+       data: writes Thing {num: Number}
+     
+     recipe R
+       h: create persistent 'some-id'
+       A
+         data: writes h`);
+
+      await emptyGenerator.collectParticleConnectionSpecs(manifest.recipes[0].particles[0]);
+      const actual = await emptyGenerator.createType(manifest.particles[0].handleConnections[0].type);
```

Comment:
Why not test the entire output with equality? It's true that you may need to update tests when you refactor, but I think that's ok. WDYT?

Reply:
I think @jasonwyatt warned that such a pattern of testing for codegen would be brittle. That's why I've tried to avoid doing so in these PRs.

Reply:
Plus, we're kind of doing a string equality in the Golens test.